src/common/include/SIRF/common/mr_image_data.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "sirf_image_data.h"
```

Comment:
would have to be `"SIRF/common/ImageData.h"`. Path is important, and filename has to be same as the class.